drivers.toml

```diff
@@ -0,0 +1,18 @@
+# If adding a new driver, please try to keep the entries alphabetized! :)
+
+[drivers.drv8825]
+name = "drv8825"
+version = "0.3.0"
+authors = [
+  "Hanno Braun <hanno@braun-embedded.com>",
+  "Jesse Braham <jesse@beta7.io>",
+]
```

Comment:
The ideal implementation would get all of this information from the `step-dir` Cargo.toml. I realize that not all the drivers have the same `authors` currently, but I don't think it's worth the effort to track them separately (anything of substance lives in `step-dir` proper).